reference-architectures/deploy-arch-ibm-slz-ocp.md

```diff
@@ -0,0 +1,138 @@
+https://test.cloud.ibm.com/docs/solution-as-code?topic=solution-as-code-naming-guidance.
+-->
+
+# OpenShift Container Platform on VPC
+{: #ocp-ra}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="Banking,FinancialSector"}
+{: toc-use-case="Cybersecurity"}
+{: toc-compliance="FedRAMP"}
+
+OpenShift Container Platform on VPC for regulated industries is a deployable architecture solution that is based on the IBM Cloud for Financial Services reference architecture. It creates secure and compliant Red Hat OpenShift Container Platform workload clusters on a Virtual Private Cloud (VPC) network.
+
+# Architecture diagram
+{: #ra-ocp-architecture-diagram}
+
+![Architecture diagram of the OpenShift Container Platform on VPC deployable architecture.](roks.drawio.svg "Architecture diagram of OpenShift Container Platform on VPC for regulated industries deployable architecture"){: caption="Figure 1. Single region architecture diagram for Red Hat OpenShift Container Platform on VPC on IBM Cloud" caption-side="bottom"}
+
+## Design requirements
+{: #ra-ocp-design-requirements}
+
+TODO: Update this heat map svg
+
+<!--
+For more information about creating a design requirements heat map image, see [Design requirements heat map](https://test.cloud.ibm.com/docs/architecture-framework?topic=architecture-framework-heat-map).
+ -->
+
+![Design requirements for Secure infrastructure on VPC for regulated industries](heat-map-deploy-arch-slz-ocp.svg "Design requirements"){: caption="Figure 2. Scope of the design requirements" caption-side="bottom"}
+
+TODO: Add the typical use case for the architecture. The use case might include the motivation for the architecture composition, business challenge, or target cloud environments.
+
+## Components
+{: #ra-ocp-components}
+
+TODO: Correct and complete the component tables. These are from PowerVS.
+
+<!--
+The listing of components and their purpose in the architecture should include what requirement the component meets, what component was chosen for this architecture, the reasons for the choice, and any alternative choices considered. These alternative choices _should_ be components or modules that are tested to swap out.
+
+If the architecture is large and includes several rows of the heat map, consider grouping each row in the design requirements heat map as an H3 section. Include a table for just that row where the caption represents that title. For example, `### Security decisions`. Otherwise, use a single table and now subsections.
+-->
+
+### VPC architecture decisions
+{: #ra-ocp-components-arch}
+
+| Requirement | Component | Reasons for choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+| * Ensure public internet connectivity  \n * Isolate most virtual instances to not be reachable directly from the public internet | Edge VPC service|Create a separate VPC service where public internet connectivity is allowed to be configured | |
```

Comment:
virtual instances or OpenShift clusters in this case?

Reply:
Updated.